src/GRA.Controllers/EventsController.cs

```diff
@@ -307,10 +350,71 @@ public IActionResult Index(EventsListViewModel model)
                 Program = model.ProgramId,
                 StartDate = startDate,
                 EndDate = endDate,
+                model.Favorites,
+                model.Visited,
                 CommunityExperiences = isCommunityExperience
             });
         }
 
+        [Authorize]
+        [HttpPost]
+        public async Task<IActionResult> UpdateSingleFavorite(int eventId, bool favorite)
+        {
+            var serviceResult = new ServiceResult();
+            try
+            {
+                var eventList = new List<Event>
+                {
+                    new Event
+                    {
+                        Id = eventId,
+                        IsFavorited = favorite
+                    }
+                };
+                serviceResult = await _activityService.UpdateFavoriteEvents(eventList);
+            }
+            catch (Exception ex)
+            {
+                var currEvent = await _eventService.GetDetails(eventId, true);
+                if (currEvent.IsCommunityExperience)
+                {
+                    _logger.LogError(ex,
+                        "Error updaing user favorite community experience: {Message}",
```

Comment:
Use {ErrorMessage} here, not {Message}

Reply:
Fixed in 0a39277094dbc091cf00e55f0f4148cfb68faa4e